weaviate/values.yaml

```diff
@@ -255,12 +255,12 @@ authentication:
 authorization:
   rbac:
     enabled: false
-    # admins:
+    # root_users:
     # - admin_user1
     # - admin_user2
-    # viewers:
-    # - viewer_user1
-    # - readonly_user1
+    # root_groups:
```

Comment:
Could you please remove the root_groups part? It seems the team do not want to expose the root_groups at this point. So it would be only:
```
rbac:
  enabled: false
  # root_users:
  # - admin_user1
  # - admin_user2
```
Thanks!